src/Components/Search/Components/Passenger.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import styled from 'styled-components';
+import { InputBox, Input, TooltipBox } from './style';
+import Button, { RgbaBtn } from '../../Button';
+
+function Passenger({
+  size,
+  selectBoxState,
+  passenger,
+  selectedSeat,
+  add,
+  substract,
+  setSeat,
+}) {
+  const [isSelectBoxOn, openSelectBox, closeSelecBox] = selectBoxState;
+
+  const handleSeat = e => {
+    const { value } = e.target;
+    e.preventDefault();
+    setSeat(value);
+  };
+
+  return (
+    <Container size={size}>
+      <InputBox iconUrl="/images/passenger.svg" onClick={e => openSelectBox(e)}>
+        <Input
+          name="passenger"
+          value={`승객 ${passenger}명, ${selectedSeat}`}
+        />
+      </InputBox>
+      {isSelectBoxOn && (
+        <SelectBox size="130%">
+          <Title as="h3">
+            인원 &amp; 좌석등급
+            <RgbaBtn
+              width="30px"
+              height="30px"
+              color="darkGray"
+              onClick={() => closeSelecBox('passenger')}
+            >
+              X
+            </RgbaBtn>
+          </Title>
+          <NumsOfPassenger>
+            <li>
+              <span>인원 수</span>
+              <Count>
+                <CountBtn color="#ced4da" outline onClick={e => substract(e)}>
+                  -
+                </CountBtn>
+                <span>{passenger}</span>
+                <CountBtn color="skyBlue" outline onClick={e => add(e)}>
+                  +
+                </CountBtn>
+              </Count>
+            </li>
+          </NumsOfPassenger>
+          <SeatList>
+            {seats.map(seat => {
+              return (
+                <li key={seat.id}>
+                  <input
+                    type="radio"
+                    name="seat"
+                    id={`seat${seat.id}`}
+                    defaultValue={seat.name}
+                    onClick={e => handleSeat(e)}
+                    //checked={seat.name === selectedSeat}
+                  />
+                  <label htmlFor={`seat${seat.id}`}>{seat.name}</label>
```

Comment:
우선 label과 input 태그의 관계를 활용하면, 즉 label 태그로 감싸주면 htmlFor와 id를 생략할 수 있습니다
```suggestion
                  <label>
                    <input
                      type="radio"
                      name="seat"
                      defaultValue={seat.name}
                      onClick={handleSeat}
                    />
                    {seat.name}
                  </label>
```

그리고 질문 주신 input 태그 동작에 관하여는 콘솔 창에 뜨는 에러메시지를 통해 힌트를 얻을 수 있습니다
```
Warning: Failed prop type: You provided a checked prop to a form field without an onChange handler. This will render a read-only field. If the field should be mutable use defaultChecked. Otherwise, set either onChange or readOnly.
```
아래와 같이 코드를 수정해주면 정상적으로 동작할 겁니다
```jsx
<input
  type="radio"
  name="seat"
  id={`seat${seat.id}`}
  value={seat.name}
  onChange={handleSeat}
  checked={seat.name === selectedSeat}
/>
```